src/lib/Libpython/module_pbs_v1.c

```diff
@@ -392,18 +393,41 @@ _pv1mod_insert_int_constants(PyObject *dict) {
 
 	/* Vnode State Constants */
 	INSERT_INT_CONSTANT("ND_FREE", INUSE_FREE);
+	INSERT_INT_CONSTANT("ND_STATE_FREE", INUSE_FREE);
 	INSERT_INT_CONSTANT("ND_OFFLINE", INUSE_OFFLINE);
+	INSERT_INT_CONSTANT("ND_STATE_OFFLINE", INUSE_OFFLINE);
 	INSERT_INT_CONSTANT("ND_DOWN", INUSE_DOWN);
+	INSERT_INT_CONSTANT("ND_STATE_DOWN", INUSE_DOWN);
+	INSERT_INT_CONSTANT("ND_STATE_DELETED", INUSE_DELETED);
 	INSERT_INT_CONSTANT("ND_STALE", INUSE_STALE);
+	INSERT_INT_CONSTANT("ND_STATE_STALE", INUSE_STALE);
 	INSERT_INT_CONSTANT("ND_JOBBUSY", INUSE_JOB);
+	INSERT_INT_CONSTANT("ND_STATE_JOBBUSY", INUSE_JOB);
 	INSERT_INT_CONSTANT("ND_JOB_EXCLUSIVE", INUSE_JOBEXCL);
+	INSERT_INT_CONSTANT("ND_STATE_JOB_EXCLUSIVE", INUSE_JOBEXCL);
 	INSERT_INT_CONSTANT("ND_RESV_EXCLUSIVE", INUSE_RESVEXCL);
+	INSERT_INT_CONSTANT("ND_STATE_RESV_EXCLUSIVE", INUSE_RESVEXCL);
 	INSERT_INT_CONSTANT("ND_BUSY", INUSE_BUSY);
+	INSERT_INT_CONSTANT("ND_STATE_BUSY", INUSE_BUSY);
 	INSERT_INT_CONSTANT("ND_STATE_UNKNOWN", INUSE_UNKNOWN);
+	INSERT_INT_CONSTANT("ND_STATE_NEEDS_HELLOSVR", INUSE_NEEDS_HELLOSVR);
+	INSERT_INT_CONSTANT("ND_STATE_INIT", INUSE_INIT);
 	INSERT_INT_CONSTANT("ND_PROV", INUSE_PROV);
+	INSERT_INT_CONSTANT("ND_STATE_PROV", INUSE_PROV);
 	INSERT_INT_CONSTANT("ND_WAIT_PROV", INUSE_WAIT_PROV);
+	INSERT_INT_CONSTANT("ND_STATE_WAIT_PROV", INUSE_WAIT_PROV);
 	INSERT_INT_CONSTANT("ND_UNRESOLVABLE", INUSE_UNRESOLVABLE);
+	INSERT_INT_CONSTANT("ND_STATE_UNRESOLVABLE", INUSE_UNRESOLVABLE);
```

Comment:
We debated this question amongst ourselves and eventually decided to duplicate the relevant constants because: (a) ND_STATE_X is more descriptive and separates node state constants from non-state ND_X constants, (b) duplicates are left in place to provide backwards compatibility, and (c) clearly identifying node state constants makes creating the reverse map from state value to name less prone to accidentally including non-state values (see end of src/modules/python/pbs/v1/_svr_types.py).

Reply:
Sure, please update the design doc to A) add the new constants and B) deprecate the old duplicated constants.  Leave the duplicates in.  Leave the duplicates in like you currently have it.

Reply:
Doc updated.

Reply:
Thank you @monkeystate